lib/time.ts

```diff
@@ -1,6 +1,6 @@
-export const millisecond = 1000
+export const millisecond = 1
 
-export const second = millisecond * 60
+export const second = millisecond * 1000
```

Comment:
Fixed definitions for second and millisecond before we accidentally deploy it with the bug